apps/elewa-group-website/src/_variables.scss

```diff
@@ -30,4 +30,10 @@
   --elewa-group-website-font-size-description: 16px;
   --elewa-group-website-font-size-description-medium: 12px;
 
+  // jobs
+  --elewa-group-website-background-color: white;
+  --elewa-group-website-font-color-h1: black;
+  --elewa-group-website-font-weight-h1: lighter;
```

Comment:
Use hexcode for color values

rename variables to just --elewa-group-website-color-white: #ffff

avoid redundancy of variables, if it already exist to create a new one.